src/ray/gcs/gcs_server/gcs_placement_group_manager.cc

```diff
@@ -147,6 +148,20 @@ void GcsPlacementGroupManager::RegisterPlacementGroup(
     }
     return;
   }
+  if (!placement_group->GetName().empty()) {
+    auto it = named_placement_groups_.find(placement_group->GetName());
+    if (it == named_placement_groups_.end()) {
+      named_placement_groups_.emplace(placement_group->GetName(),
```

Comment:
```suggestion
      RAY_CHECK(named_placement_groups_.emplace(placement_group->GetName(),...).second)
```
Let's make sure the same name is not inserted twice.

Reply:
sorry, I didn't get that, if the second pg use the exist name, it's creation will fail,  do you mean the concurrent case?